library/java/org/chromium/net/impl/CronetUrlRequestContext.java

```diff
@@ -302,6 +303,11 @@ public URLStreamHandlerFactory createURLStreamHandlerFactory() {
    */
   void onRequestDestroyed() { mActiveRequestCount.decrementAndGet(); }
 
+  @VisibleForTesting
+  public void getUrlRequestContextAdapter() {
```

Comment:
This PR does not seem to need this. If it is the case, please revert changes in this file.

Reply:
It's being used [on line 273](https://github.com/envoyproxy/envoy-mobile/pull/1765/files#diff-0ea0d6358d0189aeefee5b027be90bce715eb62c856d4d83fad6e3ed2e164cc5R273) and [line 247](https://github.com/envoyproxy/envoy-mobile/pull/1765/files#diff-0ea0d6358d0189aeefee5b027be90bce715eb62c856d4d83fad6e3ed2e164cc5R247) of the CronetURLRequestContextTest. Although the specific test cases are ignored, it's needed to maintain the infrastructure.

Reply:
I just noticed that this file was not loaded when I reviewed this PR...

But a "getter" method returning "void" is quite unorthodox. 

In this case, deviate like this is probably fine:

```
@VisibleForTesting
public boolean hasShutdown() {
  synchronized (mLock) {
    return !haveRequestContextAdapter();
  }
}
```

Reply:
Okay. I also modified the testcase to use assertTrue, instead of try/catch